spark-wrapper/spark-3.0/src/main/scala/org/apache/spark/sql/connector/write/TiDBWriteBuilder.scala

```diff
@@ -36,6 +36,7 @@ case class TiDBWriteBuilder(
           .options(tiDBOptions.parameters)
           .option(TiDBOptions.TIDB_DATABASE, tiDBOptions.database)
           .option(TiDBOptions.TIDB_TABLE, tiDBOptions.table)
+          .option(TiDBOptions.TIDB_DEDUPLICATE, "false")
```

Comment:
Why add this option, is it a bug fix?

Reply:
We want

Reply:
We want to have same behavior with TiDB.  TiDB will throw an error when there are duplicate keys. If it's `true`, we will handle it instead of throw an error.